src/mlpack/methods/hoeffding_trees/binary_numeric_split_impl.hpp

```diff
@@ -16,7 +16,7 @@
 #include "binary_numeric_split.hpp"
 
 namespace mlpack {
-namespace tree {
+namespace htree {
```

Comment:
Can I ask what the reason for this change is?

Reply:
When I add `#include <mlpack/methods/hoeffding_trees/hoeffding_tree_model.cpp>` in `inst/include/mlpack.h` it was giving some error.
```
./mlpack/methods/hoeffding_trees/information_gain.hpp:19:7: error: redefinition of ‘class mlpack::tree::InformationGain’ 
./mlpack/methods/decision_tree/information_gain.hpp:25:7: note: previous definition of ‘class mlpack::tree::InformationGain’
```

Reply:
Oh wow, I am quite surprised that that error has never been noticed before.  Do you think that we could instead rename `InformationGain` and `GiniGain` in the `hoeffding_tree/` directory to `HoeffdingInformationGain` and `HoeffdingGiniGain`, respectively?  That will cause a bit less API churn.  I can write up a quick patch if you like.

Reply:
I have renamed `InformationGain` to `HoeffdingInformationGain`, it is working great. You can take a look.
Thank You.